KsApi/models/graphql/adapters/ShippingRule+ShippingRuleFragment.swift

```diff
@@ -6,12 +6,27 @@ extension ShippingRule {
    */
   static func shippingRule(from shippingRuleFragment: GraphAPI.ShippingRuleFragment) -> ShippingRule? {
     let locationFragment = shippingRuleFragment.location.fragments.locationFragment
+    let estimatedMin = shippingRuleFragment.estimatedMin
+    let estimatedMax = shippingRuleFragment.estimatedMax
+
     guard let location = Location.location(from: locationFragment) else { return nil }
 
+    let estimatedMinMoney = Money.init(
+      amount: estimatedMin?.amount.flatMap(Double.init) ?? 0,
+      currency: Money.CurrencyCode(rawValue: estimatedMin?.currency?.rawValue ?? "")
+    )
+
+    let estimatedMaxMoney = Money.init(
+      amount: estimatedMax?.amount.flatMap(Double.init) ?? 0,
+      currency: Money.CurrencyCode(rawValue: estimatedMax?.currency?.rawValue ?? "")
+    )
+
```

Comment:
GraphAPI technically defines these as `EstimatedMin` and `EstimatedMax` types, but allows them to be `Money` types which is a struct we already have defined in our app. See [this line](https://github.com/kickstarter/ios-oss/blob/523bb45963a06229ca6bc974ebdf0498b6d3b0df/KsApi/GraphAPI.swift#L17684)